### PR TITLE
DDP-4296: Implement QA design feedback

### PR DESCRIPTION
**Pull request by fat23cat**
- Code cleaned up
- Design feedback implemented - https://docs.google.com/presentation/d/1pHRjE9rwttcMtwU5A_DRRK_UdSVkfX4MWYryxEAjQkY/edit#slide=id.g758bc0a730_1_62

- [x] Erin saw changes

### CODE REVIEW

File: ddp-workspace/projects/ddp-osteo/src/app/components/gallery/gallery.component.scss [hunk 1]
```diff
@@ -21,7 +21,7 @@
     flex-direction: column;
     justify-content: flex-start;
     grid-area: text;
-    padding: 1.25rem;
+    padding: 4rem 2rem 2rem 2rem;
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/app/components/gallery/gallery.component.scss:24]: image

File: ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 
 .dashboard-title-section {
-    padding: 2rem 0;
+    padding: 2rem 0 0 0;
     background-color: mat-color($app-theme, 350);
 }
 
+.dashboard-section {
+    margin-bottom: 10rem;
+}
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss:10]: image

File: ddp-workspace/projects/ddp-osteo/src/styles/activity.scss [hunk 3]
```diff
@@ -350,7 +355,18 @@ ddp-activity-composite-answer {
...
 .mat-error {
     font-size: 0.9rem;
+}
+
+// content question
+.ddp-content__question {
+    font-family: 'Source Serif Pro', serif;
+    font-size: 1.5rem;
+    line-height: 2.875rem;
 }
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/activity.scss]: image

File: ddp-workspace/projects/ddp-osteo/src/styles/activity.scss [hunk 4]
```diff
@@ -350,7 +355,18 @@ ddp-activity-composite-answer {
     color: mat-color($app-theme, 700);
 }
 
+.ddp-institutions-form__button {
+    padding: 1.3rem 0;
+}
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/activity.scss:387]: image

File: ddp-workspace/projects/ddp-osteo/src/styles/buttons.scss [hunk 5]
```diff
@@ -68,6 +68,10 @@
     color: mat-color($app-theme, 100);
 }
 
+.button_institutions {
+    font-size: 1.25rem;
+}
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/buttons.scss]: image

File: ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss [hunk 6]
```diff
@@ -1,10 +1,14 @@
 @import "../app-theme.scss";
 
 .dashboard-title-section {
-    padding: 2rem 0;
+    padding: 2rem 0 0 0;
     background-color: mat-color($app-theme, 350);
 }
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss:6]: image

File: ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss [hunk 7]
```diff
@@ -39,6 +43,10 @@
         line-height: 1.75rem !important;
     }
 
+    .mat-column-name {
+        flex: 0 0 20%;
+    }
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss:48]: image

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/institutions/institutionsForm.component.ts [hunk 8]
```diff
@@ -40,11 +40,13 @@ import { filter, scan, map, startWith, distinctUntilChanged, concatMap, tap } fr
...
-                (click)="addProvider()">
-            {{ block.addButtonText }}
-        </button>
+        <div class="ddp-institutions-form__button">
+            <button *ngIf="block.allowMultiple && !readonly"
+                    class="ButtonFilled ButtonFilled--white margin-5 button button_medium button_secondary button_institutions"
+                    (click)="addProvider()">
+                {{ block.addButtonText }}
+            </button>
+        <div>
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/institutions/institutionsForm.component.ts]: image

File: ddp-workspace/projects/ddp-osteo/src/styles/activity.scss [hunk 9]
```diff
@@ -319,6 +324,12 @@ select:disabled {
     }
 }
 
+.ddp-content__agreement {
+    font-family: 'Source Serif Pro', serif;
+    font-size: 1.5rem;
+    line-height: 2.875rem;
+}
+
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/activity.scss:332]: Need some backend fixes to support another style https://github.com/broadinstitute/ddp-study-server/pull/11
image

File: ddp-workspace/projects/ddp-osteo/src/styles/buttons.scss [hunk 10]
```diff
@@ -68,6 +68,10 @@
     color: mat-color($app-theme, 100);
 }
 
+.button_new-item {
+    font-size: 1.25rem;
+}
+
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/styles/buttons.scss:74]: image

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/institutions/institutionsForm.component.ts [hunk 11]
```diff
@@ -40,11 +40,13 @@ import { filter, scan, map, startWith, distinctUntilChanged, concatMap, tap } fr
...
-                (click)="addProvider()">
-            {{ block.addButtonText }}
-        </button>
+        <div class="ddp-institutions-form__button">
+            <button *ngIf="block.allowMultiple && !readonly"
+                    class="ButtonFilled ButtonFilled--white margin-5 button button_medium button_secondary button_new-item"
+                    (click)="addProvider()">
+                {{ block.addButtonText }}
+            </button>
+        <div>
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/institutions/institutionsForm.component.ts:49]: image

File: ddp-workspace/projects/ddp-osteo/src/app/components/gallery/gallery.component.scss (hunk 1 above)
Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-osteo/src/app/components/gallery/gallery.component.scss:24]: One thing I just noticed looking at carousel in mobile is that the height of the whole component  changes when you swipe through pictures. Perhaps we ought to fix the height?

File: ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss (hunk 2 above)
Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-osteo/src/styles/dashboard.scss:10]: The added spacing looks good.
Noticing that if you narrow the page, the status icon don't line up:
Screen Shot 2019-12-22 at 1 47 51 PM

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**